src/app/app.module.ts

```diff
@@ -1,7 +1,6 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
-import { AppRoutingModule } from './app-routing.module';
```

Comment:
Seems that when you decide to add another module near the 'core' module, you would add routing here. I suggest you to keep main routing file at the top level of application.

Reply:
Core module was removed and all routing now happens in app.module.